internal/cluster/cluster.go

```diff
@@ -188,22 +188,15 @@ func NewManager(cfg Config, log logrus.FieldLogger, db *gorm.DB, eventsHandler e
 }
 
 func (m *Manager) RegisterCluster(ctx context.Context, c *common.Cluster, v1Flag common.InfraEnvCreateFlag, v1ISOType models.ImageType) error {
-	err := m.registrationAPI.RegisterCluster(ctx, c, v1Flag, v1ISOType)
-	if err != nil {
-		eventgen.SendClusterRegistrationFailedEvent(ctx, m.eventsHandler, *c.ID, err.Error(), models.ClusterKindCluster)
-
-	} else {
-		eventgen.SendClusterRegisteredEvent(ctx, m.eventsHandler, *c.ID, models.ClusterKindCluster)
-	}
-	return err
+	return m.registrationAPI.RegisterCluster(ctx, c, v1Flag, v1ISOType)
```

Comment:
Is there a need to verify this won't change the behavior of automation test and their expectation?
Also, in the past we had a bug about this missing event: https://bugzilla.redhat.com/show_bug.cgi?id=2002735
Need to coordinate the change of behavior with them.

Reply:
Based on a [slack discussion with QE](https://coreos.slack.com/archives/CUPJTHQ5P/p1638696331293400), they knowledge the impact on automation,[ but they are ok with it](https://coreos.slack.com/archives/CUPJTHQ5P/p1638722040301300?thread_ts=1638696331.293400&cid=CUPJTHQ5P).